donate.html

```diff
@@ -12,6 +12,70 @@
         <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+    <script type="text/javascript">
+        	function hider(channel,no) {
+    // var x = document.getElementById('myDIV');
```

Comment:
@abhishek1995s - it will be better to refactor this JS from HTML to make it a separate file.
                             - Also remove commented code.